### PR TITLE
feat: add masa stack config

### PR DESCRIPTION
**Pull request by codding-y**

### CODE REVIEW

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/IMasaStackConfig.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+using Masa.BuildingBlocks.StackSdks.Config.Models;
```
Comment (MayueCif) [inline src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/IMasaStackConfig.cs:4]: global

File: src/Contrib/StackSdks/Tests/Masa.Contrib.StackSdks.Config.Tests/MasaStackConfigTest.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+using Masa.BuildingBlocks.StackSdks.Config;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
```
Comment (MayueCif) [inline src/Contrib/StackSdks/Tests/Masa.Contrib.StackSdks.Config.Tests/MasaStackConfigTest.cs:6]: global

**Review by MayueCif — APPROVED** [review]
LGTM @zhenlei520

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/Masa.BuildingBlocks.StackSdks.Config.csproj [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```
Comment (zhenlei520) [inline src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/Masa.BuildingBlocks.StackSdks.Config.csproj:4]: <TargetFramework>net6.0</TargetFramework>

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/Models/DbModel.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.StackSdks.Config.Models;
+
+public class DbModel
+{
+
```
Comment (zhenlei520) [inline src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/Models/DbModel.cs:8]: remove extra blank lines

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/Models/ElasticModel.cs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.StackSdks.Config.Models;
+
+public class ElasticModel
+{
+    public string ESNode { get; set; }
```
Comment (zhenlei520) [inline src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Config/Models/ElasticModel.cs:8]: ESNode ?

File: src/Contrib/StackSdks/Masa.Contrib.StackSdks.Config/Masa.Contrib.StackSdks.Config.csproj [hunk 6]
```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```
Comment (zhenlei520) [inline src/Contrib/StackSdks/Masa.Contrib.StackSdks.Config/Masa.Contrib.StackSdks.Config.csproj:4]: <TargetFramework>net6.0</TargetFramework>

File: src/Contrib/StackSdks/Tests/Masa.Contrib.StackSdks.Config.Tests/Masa.Contrib.StackSdks.Config.Tests.csproj [hunk 7]
```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```
Comment (zhenlei520) [inline src/Contrib/StackSdks/Tests/Masa.Contrib.StackSdks.Config.Tests/Masa.Contrib.StackSdks.Config.Tests.csproj:4]: net6.0

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

**Review by zhenlei520 — APPROVED** [review]
LGTM

**Pull request merged**